_posts/2017-11-09-brant-cooper-keynote-und-jede-menge-toller-speaker-und-sessions.md

```diff
@@ -0,0 +1,78 @@
+---
+layout:     post
+title:      Brant Cooper Keynote und jede Menge toller Speaker und Sessions
+authorPage: Christiane Gerigk
```

Comment:
`authorPage` should contain the page name (without the `.md`), not the title / name of the author

Reply:
In this case, it should be `christiane-gerigk` then.